tests/conftest.py

```diff
@@ -222,8 +222,7 @@ def _get_shapes() -> dict[str, GeoDataFrame]:
                 MultiPolygon(
                     [
                         Polygon(((0, 0), (0, 1), (1, 10))),
```

Comment:
this multipolygon was invalid (my_pol.is_valid was False), this created problems with sjoin